Ruby/wc/ccwc.rb

```diff
@@ -0,0 +1,47 @@
+file_content = ''
+file_name = ''
+# `ARGV` is a special global variable that holds the command-line arguments passed to a Ruby script.
+input_array = ARGV
+if (input_array.include?('-c') || 
+    input_array.include?('-l') || 
+    input_array.include?('-w') || 
+    input_array.include?('-m')) && 
```

Comment:
the results of these checks should be cached so that we don't need to do it twice.

Reply:
Even if the results are cached, we need to check it again to provide the output, right?

Reply:
updated it here - [2e9c317](https://github.com/aganapuram/CodingChallengesJC/pull/1/commits/2e9c317bad0045b332b68b823f227c770ffc2996)